woocommerce/woocommerce-gutenberg-products-block
# Allow saved HTML to render

This doesn't fix any bug I'm aware of, but we had a limitation in which passed HTML (saved via save function) would be stripped if it had tags in it, this fixes it.

It also fixes an issue in Checkbox in which spaces are removed on those type of strings.

This PR tried to update the Terms and Conditions block to use this, but it's not possible to migrate an attribute source for some reason. For any new block that has inline text, this is the recommended way of doing things.

### Testing steps

- Load this PR https://github.com/opr/newsletter-test/pull/6/
- Insert the block into checkout, save.
- Visit frontend, it should render fine.
- Edit Checkout again, edit the text and add formatting, save.
- Visit Checkout, it should be respected.
